unittests/CppInterOp/TestSharedLib1/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# TODO: Remove when library unload work correctly
```

Comment:
I do not understand why the comment. Does it mean to remove TestSharedLib1 altogether?

Reply:
Unloading libraries can hardly be implemented the way we think, so this functionality should probably be removed for now. If the unloading does work deterministically then more tests can be implemented.

Reply:
This comment is unclear what needs to be removed. Maybe remove the comment and open a github issue to track this?